packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockCore.st

```diff
@@ -0,0 +1,6 @@
+core
+mockCore
+
+	^	(TCCCore newWithTeleClient: TCTMMockTeleClient new)
+		users: self mockUsers;
+		yourself
```

Comment:
indentation seems to be off

Reply:
line 4 has a tab too much after `^`, the rest looks good in squeak. Take a look at `mockUser` for intended look